drape/glyph_manager.cpp

```diff
@@ -343,7 +343,7 @@ struct UnicodeBlock
       if (w < upperBoundWeight && w > maxWeight)
       {
         maxWeight = w;
-        index = i;
+        index = static_cast<int>(i);
```

Comment:
ASSERT that m_fontsWeight.size() is less or equal than numeric_limits<int>::max()?